patches/minecraft/net/minecraft/world/entity/Mob.java.patch

```diff
@@ -1,10 +1,14 @@
 --- a/net/minecraft/world/entity/Mob.java
 +++ b/net/minecraft/world/entity/Mob.java
-@@ -206,6 +_,7 @@
+@@ -206,6 +_,11 @@
  
     public void m_6710_(@Nullable LivingEntity p_21544_) {
        this.f_21362_ = p_21544_;
```

Comment:
Does not match merged PR. Line was removed.

Reply:
Fixed by 8c71d59dbd73bf58841c6cf189cd307b29378e9e.